src/Vezeeta.Domain/Entities/Appointment.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vezeeta.Domain.Enums;
+
+namespace Vezeeta.Domain.Entities
+{
+    public class Appointment
```

Comment:
For all entities, you should be inheriting from `BaseEntity<T>` where T is the datatype of the ID for your entity. This is to follow the DDD in a simple fashion. This edit is to be applied in all Entities except the `ApplicationUser`